### PR TITLE
Update kong.conf.default

### PR DESCRIPTION
**Pull request by DeclanPaulKeane**
### Summary

This is my first time doing this, so apologies if I make a mistake. The path for status_listen is missing and the docs need to be updated.

### Checklist

- [ x] The Pull Request has tests
- [ x] There's an entry in the CHANGELOG
- [ x] There is a user-facing docs PR against https://github.com/Kong/docs.konghq.com - PUT DOCS PR HERE

### CODE REVIEW

**Review by lena-larionova — APPROVED** [review]
LGTM, will pull it into the docs once this is merged.

**Review by hbagdi — CHANGES_REQUESTED** [review]
This change doesn't look right.
There are more endpoints available than /status on the status endpoint.
@Guaris is working on a spec that lays them out.

**Comment by hbagdi** [comment]
Here is the spec: https://developer.konghq.com/spec/9542436a-58d1-4522-a00b-32125b5940f0/5feb007a-348b-486d-8d06-92f615471d22

Please re-open if needed.

**Pull request closed without merging**
